docker-compose/config/common.libsonnet

```diff
@@ -36,5 +36,12 @@
   browserUrl: 'http://localhost:7984',
   httpListenAddress: ':80',
   maximumMessageSizeBytes: 16 * 1024 * 1024,
-  global: { diagnosticsHttpListenAddress: ':9980' },
+  global: {
+    diagnosticsHttpServer: {
+      listenAddress: '127.0.0.1:12345',
```

Comment:
Please keep the listen address the same as before. We probably do pod health checking on it? Or at least, I hope so.